src/config.h

```diff
@@ -83,7 +83,7 @@ struct RviConfig {
 
 struct TlsConfig {
   TlsConfig()
-      : certificates_directory("/tmp/aktualizr"),
+      : certificates_directory("./sota_prov"),
```

Comment:
Is this a change we want to make? I think there is some sense to keeping the default as an absolute path.